src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -1,21 +1,43 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import data from '../data.json';
+import defaultLinkData from '../mega-menu-link-data.json';
+import authenticatedUserLinkData from '../mega-menu-link-data-for-authenticated-users.json';
 import { togglePanelOpen, toggleMobileDisplayHidden } from '../actions';
+import { isLoggedIn } from '../../../user/selectors';
 
 import MegaMenu from '@department-of-veterans-affairs/formation/MegaMenu';
 
 // const SESSION_REFRESH_INTERVAL_MINUTES = 45;
 
+export function flagCurrentPageInTopLevelLinks(links = [], pathName = window.location.pathname) {
+  return links.map(link => {
+    return pathName.endsWith(link.href) ? { ...link, currentPage: true } : link;
+  });
+}
+
+export function maybeMergeAuthenticatedLinkData(loggedIn, authenticatedLinks = authenticatedUserLinkData, defaultLinks = defaultLinkData) {
```

Comment:
How about renaming to something like `getAuthorizedLinkData` to indicate that it will determine what links are relevant for the user, depending on whether or not they are authenticated and therefore authorized?

Reply:
I'll rename the file to use `authorized` as well- I'm not familiar with our level of access nomenclature. authenticated, authorized, identified, etc

Reply:
Yes, AFAIK there is not a standard nomenclature for this.